ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -410,7 +411,11 @@ private EndpointResponse handleQuery(final KsqlSecurityContext securityContext,
             configProperties)) {
 
       final ImmutableAnalysis analysis = ksqlEngine
-          .analyzeQueryWithNoOutputTopic(statement.getStatement(), statement.getStatementText());
+          .analyzeQueryWithNoOutputTopic(
+              statement.getStatement(),
+              statement.getStatementText(),
+              request.getConfigOverrides()
```

Comment:
```suggestion
              configProperties
```
Let's be consistent with above.